packages/react-core/src/components/Select/Select.tsx

```diff
@@ -143,6 +143,8 @@ export interface SelectProps
   maxHeight?: string | number;
   /** Icon element to render inside the select toggle */
   toggleIcon?: React.ReactElement;
+  /** Custom icon for the dropdown replacing the CaredDownIcon */
```

Comment:
typo though you might want to revisit and be consistent with the dropdown's `toggleIndicator` prop description? 

```suggestion
  /** Custom icon for the dropdown replacing the CaretDownIcon */
```